wgpu-hal/src/vulkan/device.rs

```diff
@@ -535,6 +535,40 @@ impl super::Device {
             config: config.clone(),
         })
     }
+
+    pub fn image_raw_flags(desc: &crate::TextureDescriptor) -> vk::ImageCreateFlags {
```

Comment:
actually, how about not exposing this at all? I don't think XR cares about cube views, anyway. This function is a hack, and hacks should ideally stay private